examples/babylonjs/index.js

```diff
@@ -44,6 +44,10 @@ var createScene = function(engine) {
     var guiBoundingBox = gui.add(window, 'BOUNDING_BOX').name('Bounding Box');
     var guiDebug = gui.add(window, 'DEBUG').name('Debug');
 
+    BABYLON.SceneLoader.OnPluginActivatedObservable.add(function (plugin) {

```

Comment:
You can use `addOnce` instead instead of passing in all of the `undefined` with a `true` at the end.

Reply:
Thanks.  I'll copy that back to VSCode too.